bgpd/bgp_mplsvpn.c

```diff
@@ -964,57 +964,6 @@ void transpose_sid(struct in6_addr *sid, uint32_t label, uint8_t offset,
 	}
 }
 
-static bool labels_same(struct bgp_path_info *bpi, mpls_label_t *label,
```

Comment:
You remove a function that you have added in a previous commit. Use the final function in the initial commit.